modules/shortcodes/wordads.php

```diff
@@ -53,14 +57,41 @@ static function wordads_shortcode_html( $atts, $content = '' ) {
 			return '<div>' . __( 'The WordAds module is not active', 'jetpack' ) . '</div>';
 		}
 
-		$html = '<div class="jetpack-wordad" itemscope itemtype="https://schema.org/WPAdBlock">';
-
-		$html .= '</div>';
-
+		$html = '<div class="jetpack-wordad" itemscope itemtype="https://schema.org/WPAdBlock"></div>';
 		$html = $wordads->insert_inline_ad( $html );
 
 		return $html;
 	}
+
+	public static function gutenblock_render( $attr ) {
+		global $wordads;
+
+		/** This filter is already documented in modules/wordads/wordads.php `insert_ad()` */
+		if ( is_feed() || apply_filters( 'wordads_inpost_disable', false ) ) {
+			return '';
+		}
+
+		if ( $wordads->option( 'wordads_house' ) ) {
+			return $wordads->get_ad( 'inline', 'house' );
+		}
+
+		$section_id = 0 === $wordads->params->blog_id ? WORDADS_API_TEST_ID : $wordads->params->blog_id . '6'; // 6 is to keep track of gutenblock ads
```

Comment:
I'm not sure how this will play out, could we add parens around the ternary or the concat `.` to make it clear what the order of operations is and what to expect?

Reply:
I broke it up onto different lines for clarity (and because it was getting pretty long).